app/controllers/landing_page_controller.rb

```diff
@@ -5,8 +5,14 @@ class LandingPageController < ContentItemsController
 
   # SCAFFOLDING: can be removed when basic content items are available
   # from content-store
-  def request_content_item(_base_path)
-    GdsApi.content_store.content_item(request.path).to_h
+  def content_item
+    @content_item ||= ContentItemFactory.build(old_scaffolding_content_item)
+  end
+
+  # SCAFFOLDING: can be removed when basic content items are available
+  # from content-store
+  def old_scaffolding_content_item
+    ContentItemLoaderGdsApi.content_store.content_item(request.path).to_h
```

Comment:
This doesn't look right. There isn't actually a class called `ContentItemLoaderGdsApi` is there?

The tests are passing, so either this is fine somehow, or this bit of code is not being tested.

Reply:
Yeah, not sure what was going on here. I've fixed it now!